prow/plugins/lifecycle/reopen.go

```diff
@@ -73,8 +72,14 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) e
 		return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, resp))
 	}
 
+	if warn {
+		if err := deprecate(gc, "reopen", org, repo, number, e); err != nil {
+			return err
+		}
+	}
+
 	if e.IsPR {
-		log.Info("Re-opening PR.")
+		log.Infof("/reopen PR %s/%s#%d", org, repo, number)
```

Comment:
The org, repo, and number are already included as fields in the logger given to event handlers if the event is associated with a specific issue or PR so you don't need to include them in any of these logging calls.

Reply:
Done